ruff.toml

```diff
@@ -0,0 +1,18 @@
+exclude = [
+    "input",
+    "__pycache__",
+    ".venv",
+]
+
+target-version = "py39"
+line-length = 88
```

Comment:
I'm also curious why 88 rather than keeping 120? The rationale for 120 originally is that more people are working in screens that are larger than original IBM machine (though actually thinking to that example, isn't that 80 columns?)